apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -84,15 +79,14 @@ const Invalid = () => (
 		className="is-warning"
 		title={ __( 'Please fill in the required fields' ) }
 		subTitle={ __(
-			'Title, Org, and Repository are required. Select this block to open the form and fill them.'
+			'Org, and Repository are required. Select this block to open the form and fill them.'
 		) }
-		icon="⚠"
 	/>
 );
 
 registerBlockType( 'a8c/github-issue-template', {
```

Comment:
👍🏻

Reply:
Actually, while I like the idea, I think it's best if we leave this out for another iteration. This will require splitting the artifacts into more files ideally and I wouldn't want to refactor to this level at this point, considering this has been going for a while already.